src/pages/Pool/index.tsx

```diff
@@ -112,7 +107,7 @@ export default function Pool() {
     trackedTokenPairs.length === 0 ||
     dxSwapPairs?.length < liquidityTokensWithBalances.length ||
     dxSwapPairs?.some(DXSwapPair => !DXSwapPair)
-
+  console.log(dxSwapPairs)
```

Comment:
This needs to be removed

Reply:
https://github.com/levelkdev/dxswap-dapp/commit/bc86d3290120bd5441da0d778b57578396d79e98